app/tasks/mail_error_task.py

```
@@ -24,12 +23,35 @@
     Mail Error Task. This handles tasks that have failed to deliver messages
     """
-    log.info(f"Received Data={data}")
+    bound_log = log.bind(request_id=request_id, celery_task_id=getattr(self.request, 'id', None))
+    bound_log.info("Received failed message for inspection", data=data)
+
+    try:
+        from app.metrics import email_error_tasks
+
+        email_error_tasks.inc()
+    except Exception:
+        pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)